ios/Classes/SwiftAirshipPlugin.swift

```diff
@@ -4,18 +4,20 @@ import Airship
 
 public class SwiftAirshipPlugin: NSObject, FlutterPlugin, UARegistrationDelegate,
 UADeepLinkDelegate, UAPushNotificationDelegate {
-    private let eventNameKey = "event_name";
-    private let eventValueKey = "event_value";
-    private let propertiesKey = "properties";
-    private let transactionIDKey = "transaction_id";
-    private let interactionIDKey = "interaction_id";
-    private let interactionTypeKey = "interaction_type";
-    private let tagOperationGroupName = "group";
-    private let tagOperationType = "operationType";
-    private let tagOperationTags = "tags";
-    private let tagOperationAdd = "add";
-    private let tagOperationRemove = "remove";
-    private let tagOperationSet = "set";
+    private let pluginVersion = "2.0.0"
```

Comment:
we are setting this in two places

Reply:
nah, I moved this into its own class and forgot about this, removed